app/templates/releases/index.hbs

```diff
@@ -25,7 +25,29 @@
 
 <h2>Upcoming releases</h2>
 
-<ReleaseTimeline class="hide-on-mobile" @project={{emberBetaProject}} />
+The Ember project aims to release approximately every 6 weeks.
+For a very rough estimate of the timeline for the next release,
+you can look at the date of the most recent
+<a href="https://blog.emberjs.com/tags/releases.html">
+  official release blog post
+</a>
+and count forward by six weeks.
+The release timelines fluctuate based on team availability
+across all projects, holidays, and for extended testing
+before a release.
```

Comment:
This sounds like a repeat of the above. Do you mean to say that the actual releases aren't on the dot because of team availability?

Reply:
More or less, yeah. I can simplify this some!